docs/helm/helm-chart-verification-via-console.md

```diff
@@ -0,0 +1,45 @@
+
+### Table 1: Helm chart check types
+
+Helm chart checks are categorized into the following types:
+
+| Check type | Description
+|---|---
+| Mandatory | Checks are required to pass and be successful for certification.
+| Recommended | Checks are about to become mandatory; we recommend fixing any check failures.
+| Optional | Checks are ready for customer testing. Checks can fail and still pass the verification for certification.
+| Experimental | New checks introduced for testing purposes or beta versions.
+> **_NOTE:_**  The current release of the chart-verifier includes only the mandatory and optional type of checks.
+
+## Default set of checks for a Helm chart
+
+The `/api/helm/verify` Helm chart-verifier API integrates the chart-verifier CLI tool on Openshift Developer Console (ODC). This API provides users more insight into whether the Helm charts that they are targeting run well on OpenShift. The following checks are run when the user needs to verify their Helm chart from the console:
+
+### Table 2: Helm chart default checks
+
+| Checks | Description |
+|---|---|
+| IsHelmV3 | Checks that the given `uri` points to a Helm v3 chart. |
+| HasReadme | Checks that the Helm chart contains the `README.md` file. |
+| ContainsTest | Checks that the Helm chart contains at least one test file. |
+| HasKubeVersion | Checks that the `Chart.yaml` file of the Helm chart includes the `kubeVersion` field (v1.0) and is a valid semantic version (v1.1). |
+| ContainsValuesSchema | Checks that the Helm chart contains a JSON schema file (`values.schema.json`) to validate the `values.yaml` file in the chart. |
+| NotContainsCRDs | Checks that the Helm chart does not include custom resource definitions (CRDs). |
+| NotContainCsiObjects |  Checks that the Helm chart does not include Container Storage Interface (CSI) objects. |
+| ImagesAreCertified | Checks that the images referenced by the Helm chart are Red Hat-certified .|
+|  HelmLint | Checks that the chart is well formed by running the `helm lint` command. |
+| ContainsValues | Checks that the Helm chart contains the `values`[¹](https://github.com/redhat-certification/chart-verifier/blob/main/docs/helm-chart-checks.md#-for-more-information-on-the-values-file-see-values-and-best-practices-for-using-values) file. |
+| RequiredAnnotationsPresent |Checks that the Helm chart contains the annotation: ```charts.openshift.io/name```. |
+
+A chart that goes through the certification process verifies that the Helm chart is compliant with the predefined set of independent checks and is recommended for use.
+
+The output of the chart-verifier API is a JSON-encoded [response ](https://github.com/redhat-certification/chart-verifier/blob/main/pkg/chartverifier/report/types.go#L12-18).
```

Comment:
I think we should talk about profiles here, I would create a profile for console under chart verifier. We default to that if api client does not specify the profile.